analyses/snv-callers/util/wrangle_functions.R

```diff
@@ -96,7 +95,7 @@ maf_to_granges <- function(maf_df) {
       start = maf_df$Start_Position,
       end = maf_df$End_Position
     ),
-    strand = maf_df$Strand,
+    # strand = maf_df$Strand,
```

Comment:
I might put a brief comment here that contains some of the information that is included in the text of the PR.